README.md

```diff
@@ -93,11 +93,12 @@ The output is written to `Filters/filter.txt`.
 ###  Exception
 #### This list is used to exclude domains that cause serious site breakages
 
+* [Exclusions rules](https://github.com/AdguardTeam/AdGuardSDNSFilter/blob/master/Filters/exclusions.txt)
```

Comment:
`Exclusions rules` => `Exclusion rules`, or `Exclusions`(and `Exceptions` below)